ads/google/imaVideo.js

```diff
@@ -248,17 +257,38 @@ export function imaVideo(global, data) {
     'bottom': '0px',
     'width': '100%',
     'height': '100px',
+    'background-color': 'rgba(7, 20, 30, .7)',
+    'background':
+      'linear-gradient(0, rgba(7, 20, 30, .7) 0%, rgba(7, 20, 30, 0) 100%)',
     'box-sizing': 'border-box',
     'padding': '10px',
     'padding-top': '60px',
-    'background-image': `url(${controlsBg})`,
-    'background-position': 'bottom',
     'color': 'white',
     'display': 'none',
     'justify-content': 'center',
     'align-items': 'center',
     'user-select': 'none',
+    'z-index': '1',
   });
+  // Ad progress
+  countdownWrapperDiv = global.document.createElement('div');
+  countdownWrapperDiv.id = 'ima-countdown';
+  setStyles(countdownWrapperDiv, {
+    'align-items': 'center',
+    'box-sizing': 'border-box',
+    'display': 'none',
+    'flex-grow': '1',
+    'font-family': 'Helvetica, Arial, Sans-serif',
```

Comment:
Do we need this?

Reply:
This style was being set in `timeDiv` so I also set it in `countdownWrapperDiv` for consistency.  However, now I've moved this style to their shared parent, `controlsDiv`, and have verified that both `timeDiv` and `countdownWrapperDiv` inherit the font-family style.

Reply:
Thank you! 😄